docs/fastdds/getting_started/simple_python_app/includes/dependencies.rst

```diff
@@ -0,0 +1,34 @@
+Import linked libraries and its dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The DDS application requires the Fast DDS, Fast CDR and Fast DDS Python bindings libraries.
```

Comment:
Be more specific about Fast DDS Python bindings.
There is no place in this documentation where the repo is linked or any instruction on how to use it. At least add here the link to the repo.